### PR TITLE
changing default severity

### LINKED ISSUE #368: Notification as Default Severity
There are several feedbacks coming from distinct stakeholders related to the code pal severities, and in a short, I agree with them.

Code pal is a tool based on the Clean ABAP style guide. It should not define what is an error or what is a warning, but so notify the developer that something was identified which is not compliant with the style guide. It is just automation.

It is the development team's responsibility, together with their Architect and Quality Engineer, to decide if a rule is a warning or an error. 

Therefore, I propose to set all the Checks as notifications by default.

### PR DESCRIPTION
**Pull request by lucasborin**
Closes #368

### CODE REVIEW

File: src/checks/y_check_branch_coverage.clas.abap [hunk 1]
```diff
@@ -16,9 +16,8 @@ CLASS y_check_branch_coverage IMPLEMENTATION.
   METHOD constructor.
     super->constructor( ).
 
-    version = '0001'.
+    version = '0002'.
```
Comment (lucasborin) [inline src/checks/y_check_branch_coverage.clas.abap:19]: version = version + 1.

File: src/checks/y_check_procedure_coverage.clas.abap [hunk 2]
```diff
@@ -16,9 +16,8 @@ CLASS y_check_procedure_coverage IMPLEMENTATION.
   METHOD constructor.
     super->constructor( ).
 
-    version = '0001'.
+    version = '0002'.
```
Comment (lucasborin) [inline src/checks/y_check_procedure_coverage.clas.abap:19]: version = version + 1.

File: src/checks/y_check_returning_name.clas.abap [hunk 3]
```diff
@@ -20,10 +20,9 @@ CLASS y_check_returning_name IMPLEMENTATION.
     settings-pseudo_comment = '"#EC RET_NAME' ##NO_TEXT.
     settings-disable_threshold_selection = abap_true.
     settings-threshold = 0.
-    settings-prio = c_warning.
     settings-documentation = |{ c_docs_path-checks }returning-name.md|.
 
-    version = '001'.
+    version = version + 1.
```
Comment (lucasborin) [inline src/checks/y_check_returning_name.clas.abap:25]: move it to top

File: src/checks/y_check_statement_coverage.clas.abap [hunk 4]
```diff
@@ -16,16 +16,15 @@ CLASS y_check_statement_coverage IMPLEMENTATION.
...
-
-    settings-prio = c_note.
     settings-threshold = 60.
     settings-is_threshold_reversed = abap_true.
     settings-disable_on_prodcode_selection = abap_true.
     settings-disable_on_testcode_selection = abap_true.
     settings-apply_on_test_code = abap_false.
     settings-documentation = |{ c_docs_path-checks }unit-test-coverages.md|.
 
+    version = version + 1.
```
Comment (lucasborin) [inline src/checks/y_check_statement_coverage.clas.abap:26]: move to top

**Comment by lucasborin** [comment]
it corrupts the variant 😢

**Comment by lucasborin** [comment]
"corrupts" => it forces the user to review the new customizing on each check. To avoid manual efforts in the upgrade, let's keep the same Check version.

**Review by estevao-schultz-neto-SAP — CHANGES_REQUESTED** [review]
I've just adjusted minor things in the text/docu.

File: pages/how-to-configure.md [hunk 5]
```diff
@@ -4,9 +4,9 @@
 
 ## How to Configure
 
-💡 The tool is compatible with the SAP Code Inspector framework (`SCI` / `SCII`).
+💡 The tool is compatible with the SAP Code Inspector framework (`SCI` / `SCII`) and you can launch it using your own SCI/SCII variant(s). Alternatively, you may also use the tool by means of defining Profiles as described bellow.
 
-⚠️The below documentation is valid for our own `Profile Management Tool` feature only.
+⚠️The below documentation is valid for our own `Profile Management Tool` (transaction: `Y_CODE_PAL_PROFILE`) feature only.
```
Comment (lucasborin) [inline pages/how-to-configure.md:9]: The transaction is not duplicated (line 9 and 22). How about if we link to the `Profile Management Tool` page?

File: pages/how-to-configure.md [hunk 6]
```diff
@@ -4,9 +4,9 @@
 
 ## How to Configure
 
-💡 The tool is compatible with the SAP Code Inspector framework (`SCI` / `SCII`).
+💡 The tool is compatible with the SAP Code Inspector framework (`SCI` / `SCII`) and you can launch it using your own SCI/SCII variant(s). Alternatively, you may also use the tool by means of defining Profiles as described bellow.
```
Comment (lucasborin) [inline pages/how-to-configure.md:7]: @estevao-schultz-neto-SAP: it looks redundant as we mentioned the transactions twice.

**Review by estevao-schultz-neto-SAP — APPROVED** [review]

**Pull request merged**